src/cli/cmd/ping.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from cli.config import ProjectConfiguration
+from cli.utils.yaml import write_file
+ 
+from core.common.drivers.factory import load_driver
+from core.common.drivers.factory import load_config
+from .base import BaseCmd
+
+from sqlalchemy.exc import OperationalError,DBAPIError
+class PingCmd(BaseCmd):
+  def run(self):
+      source = self.args
```

Comment:
Consider checking for source attribute and then assigning accordingly. 

Logging.warn if none provided that we're using default